Vocabulary_Count.ipynb

```diff
@@ -0,0 +1,165 @@
+{
```

Comment:
<p>Um. Confusion is me. Where are we using data_lst again? Shouldn't this be appended to text?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/Hinglish/pull/55/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>